### DISCUSSION TITLE
how to add a caption to a latex equation?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#6160] [Q&A]
### Description

May I ask how to add a caption to a latex equation like the following? 

I'm unable to find a reference in the [quarto doc on latex figure](https://quarto.org/docs/authoring/figures.html#latex-figures). 

<img width="537" alt="Screenshot 2023-07-10 at 11 06 39 PM" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/26fe07f3-7c90-49f6-b19e-fb481f59b9f2">

**Comment by mcanouil** [comment]
If you look for equation related things in the figure section of the documentation, you are very unlikely to find what you want.
See <https://quarto.org/docs/authoring/cross-references.html#equations> for what's currently possible.

**Comment by mcanouil** [comment]
Additionally:

- <https://github.com/jgm/pandoc/issues/8145>
- <https://github.com/quarto-dev/quarto-cli/issues/4944> (work ongoing here should/will allow more flexibility that is currently not allowed in Pandoc/Quarto)

**Comment by s2t2** [comment]
So how to answer this question, though? The links don't seem to provide a working example of how to add a caption to an equation.

This might be a reasonable workaround for adding a caption to an equation:

```
::: {.equation #eq:simple-equation}
$$
\frac{x+2}{3x+6} = \frac{1(x+2)}{3(x+2)} = \frac{1}{3}
$$
*Equation 1: Simplification of the equation*
:::
```

<img width="615" alt="Screenshot 2024-09-12 at 3 08 43 PM" src="https://github.com/user-attachments/assets/f269a5e3-1e12-4f3c-8958-81af574dfedc">

**Reply by cscheid** [reply]
> The links don't seem to provide a working example of how to add a caption to an equation.

That's because it's a feature that Quarto doesn't support.

**Reply by mcanouil** [reply]
Use a custom cross-reference, see <https://quarto.org/docs/authoring/cross-references-custom.html#pdf-output>.


```qmd
---
title: "Quarto Playground"
format: pdf
crossref:
  custom:
    - kind: float
      reference-prefix: Equation
      key: eqn
      latex-env: eqn
---

::: {#eqn-simplify}
$$
\frac{x+2}{3x+6} = \frac{1(x+2)}{3(x+2)} = \frac{1}{3}
$$

Simplification of the equation
:::
```
